web/src/test/resources/com/epam/rft/atsy/cucumber/CandidateCreation.feature

```diff
@@ -0,0 +1,115 @@
+Feature: candidate creation test
+
+  Background:
+    Given The user signed in
+
+  Scenario Outline: user can't create new candidate because of empty required field
+
+    Given the user is on the Candidate creation page
+    And the candidate details are empty
+    And the user enters name "name"
+    And the user enters e-mail address "email@atsy.com"
+    And the user enters phone number "+36301234567"
+    And the user enters the place where the candidate has heard about the company "place"
+    And the user enters the language level "4"
+    And the user clears field "<field>"
+    When the user clicks on the "Mentés" button
+    Then a "<message>" message is shown under the "<field>" field
+
+    Examples:
+      | field | message                                    |
+      | name  | A jelentkező nevét kötelező megadni!       |
+      | email | A jelentkező email címét kötelező megadni! |
+
+  Scenario: user can't create new candidate because of duplication based on same email address
+
+    Given other candidates data collected
```

Comment:
The following scenarios are really similar, they can be done using scenario outlines.